### ISSUE TITLE
[BUG] "Unlabeled, Double tap to activate" / "Button, Double tap to activate" needs changed throughout app and web

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5572]
**Describe the bug**
Each button and interactable element needs labeled for assistive technology. There are several instances of  "Unlabeled, Double tap to activate" / "Button, Double tap to activate" throughout the app and web. I cannot open an individual bug for each of them. It is not a good use of time. Please CTRL + F to find these instances in the code and label items accordingly. 

**Expected behavior**
Any button or interactable element needs to be labeled for assistive technology users.

Should function the same across all platforms and devices. 

These are only a couple of examples, please CTRL + F to find all instances and fix: 

![Image](https://github.com/user-attachments/assets/30ebfa11-e983-4a87-be02-a5e6660928ce)

![Image](https://github.com/user-attachments/assets/3f493346-4a77-479a-96d3-9c7e25336f02)

**Additional context**

**Comment by acovher2** [comment]
As mentioned in bug #5573 This is affecting Android currently. Why are the button labels not announced by TalkBack?

**Comment by TodorBachvarov** [comment]
@acovher2, the close button in the Event Home Panel was fixed in version 7.3.34. Regarding the buttons in the Edit Profile Panel (the second screenshot) seems to have labels, so I'm not able to reproduce this part. If you still reproduce it please make sure that you are using the mentioned version (a video of the issue may help if still reproducible)
